herbert_core/utilities/classes/@Hashing/Hashing.m

```diff
@@ -0,0 +1,76 @@
+classdef Hashing
+    %HASHING Independent class providing hash creation functions
```

Comment:
Is it a class or a mixin?

Reply:
It is a class providing a static member. It is not intended presently that unique_objects_container derive from it. See unique_objects_container.hashify(...)